src/JsonRpc/JsonRpcServerServiceCollectionExtensions.cs

```diff
@@ -47,13 +47,22 @@ public static class JsonRpcServerServiceCollectionExtensions
                                 .Type<IScheduler>(serviceKey: nameof(options.OutputScheduler)),
                 reuse: Reuse.Singleton
             );
+
+            container.Register<RequestInvokerOptions>(
+                made: new Made.TypedMade<RequestInvokerOptions>().Parameters
+                                                      .Type<TimeSpan>(serviceKey: nameof(options.MaximumRequestTimeout))
+                                                      .Type<bool>(serviceKey: nameof(options.SupportsContentModified))
+                                                      .Name("concurrency", serviceKey: nameof(options.Concurrency)),
+                reuse: Reuse.Singleton);
+
+            if (!container.IsRegistered<RequestInvoker>())
+            {
+                container.Register<RequestInvoker, DefaultRequestInvoker>(Reuse.Singleton);
```

Comment:
This is missing `.Type<IScheduler>(serviceKey: nameof(options.InputScheduler))`

Reply:
ooo, out of curiosity why not have that be registered by default?